### PR TITLE
Initial Code for maplibre-gl-compare ...

### PR DESCRIPTION
**Pull request by astridx**
... based on https://github.com/mapbox/mapbox-gl-compare

### CODE REVIEW

File: API.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+[8]: #fire
+
+[9]: #parameters-3
+
+[10]: #off
+
+[11]: #parameters-4
+
+[12]: https://maplibre.org/maplibre-gl-js-docs/plugins/
```
Comment (astridx) [inline API.md:105]: Perhaps there is a more precise address here?

File: README.md [hunk 2]
```diff
@@ -1 +1,274 @@
...
+<link href="https://unpkg.com/maplibre-gl@1.14.0/dist/maplibre-gl.css" rel='stylesheet' />
+<link href="maplibre-gl-compare.css" rel='stylesheet' />
+
+<script src="https://unpkg.com/maplibre-gl@1.14.0/dist/maplibre-gl.js"></script>
+<script src="maplibre-gl-compare.js"></script>
+```
+
+### Node 
+
+
```
Comment (astridx) [inline README.md:259]: Should an installation via npm be possible? Who should take care of this?

File: README.md [hunk 3]
```diff
@@ -1 +1,274 @@
...
+//Remove - this will remove the compare control from the DOM and stop synchronizing the two maps.
+compare.remove();
+```
+
+## Live examples
+
+[Compare hybrid with streets](https://rawcdn.githack.com/astridx/astridx.github.io/a9d7297a4fe1e3a4d7ebeb1e4e662fd1339ef3b5/maplibreexamples/plugins/maplibre-gl-compare-swipe-between-maps.html)
+
+[Compare swisstopo with demotiles](https://rawcdn.githack.com/astridx/astridx.github.io/a9d7297a4fe1e3a4d7ebeb1e4e662fd1339ef3b5/maplibreexamples/plugins/maplibre-gl-compare-swipe-between-maps_.html)
```
Comment (astridx) [inline README.md:240]: Are the links OK? I use files in my repo.

**Comment by wipfli** [comment]
Looks good to me

File: README.md (hunk 3 above)
Comment (wipfli) [reply README.md:240]: Yes, thanks for the examples.

File: README.md (hunk 2 above)
Comment (wipfli) [reply README.md:259]: We want to publish also https://github.com/maplibre/maplibre-gl-leaflet/ on NPM. So once we know how to do it with the leaflet plugin, we can also do the same here.

File: API.md (hunk 1 above)
Comment (wipfli) [reply API.md:105]: Looks good to me

**Pull request merged**